apps/admin/oauth2-proxy/oauth2-proxy.yaml

```diff
@@ -18,6 +18,9 @@ spec:
   interval: 1m
   values:
     extraArgs:
+      cookie-domain: ".demo.platform.hmcts.net"
+      redirect-url: https://oauth-proxy.demo.platform.hmcts.net/oauth-proxy/callback
+      whitelist-domain: ".demo.platform.hmcts.net"
```

Comment:
Remove these again. They will get picked up from the cluster specific file when deployed via flux